### ISSUE TITLE
Version 2.X.X support

### ISSUE THREAD

**Issue by kierun** [bueltge/Markdown-for-MantisBT#11]
Is there any plans to support 2.X.X?

**Comment by bueltge** [comment]
Yes, I plan that it also work at the 2.* But currently is my time frame really small.
Have you tried this version in 2.*, have you problems, issues? My installs are on 1.2 and 1.3, live systems. A update is in planing, bot no appointment in the current planing.

**Comment by kierun** [comment]
Yes, I did. On beta 1, I did get it working by adding this to `Markdown.php`. 

```
$this->requires    = array(
  'MantisCore'           => '1.2.0, 1.3.0, 2.0.0',
  'MantisCoreFormatting' => '1.0a',
);
```

When I tried that on 2.0.0 release, I get a blank configuration page. I assumed it was not working but testing some markdown in notes seem to work just fine. The configuration page not showing does not seem to matter although since I moved from 1.X, I assume that the settings are fine in the database. I just cannot change them!

I did add a dependency to `MantisCoreFormatting` to 2.0.0.
